app/lib/common/widgets/attachments/attachment_section.dart

```diff
@@ -64,6 +73,127 @@ class AttachmentSectionWidget extends ConsumerWidget {
     );
   }
 
+  Widget _buildAttachmentItem(
+    BuildContext context,
+    Attachment item,
+    bool? canRedact,
```

Comment:
No need to pass `canRedact` as it can be directly accessible from class variable.

Reply:
You're right. Thanks for pointing it out!

Reply:
Fixed by e704014.